iree/compiler/Dialect/Shape/IR/Builders.cpp

```diff
@@ -126,7 +126,8 @@ LogicalResult getRankedDimsFromRankedShape(Location loc, Value rsValue,
                                            SmallVectorImpl<Value> &outDims,
                                            OpBuilder &builder) {
   Operation *op = rsValue.getDefiningOp();
-  if (llvm::isa<MakeRankedShapeOp>(op) || llvm::isa<ConstRankedShapeOp>(op)) {
+  if (op &&
```

Comment:
I think we can use `dyn_cast_or_null<XXX>(op)`, so we don't need to check whether op is null.

Reply:
This is just isa in, not a dyncast, so probably fine/preferable to have the null check first.